### PR TITLE
Added visual radius

### PR DESCRIPTION
**Pull request by mjrmua**

### CODE REVIEW

**Comment by mjrmua** [comment]
is it finally working?

**Comment by stankiewiczm** [comment]
Looks like it's ready for review :+1:

File: run [hunk 1]
```diff
@@ -1 +0,0 @@
-python -m http.server 8080
```
Comment (stankiewiczm) [inline run]: Fairly sure we still need this file to run locally

File: src/visibility.js [hunk 2]
```diff
@@ -44,4 +44,4 @@ const lineOfSightTest = world => fromPoint => toPoint => {
         .map(v => ({ ...v, radius: 0.5 }))
         .filter(circleIntersectsLine(line))
         .length == 0;
-}
```
Comment (stankiewiczm) [inline src/visibility.js]: Line-ending wars :P

File: src/game.js [hunk 3]
```diff
@@ -227,6 +242,7 @@ function update_inventory(item) {
 }
 
 var possibleDestinations = {
+    "#": moveToWall,
```
Comment (stankiewiczm) [inline src/game.js]: How is it possible to run into an item that's not in your line of sight?  We only move to adjacent squares.

Comment (samperumal) [reply src/game.js]: I assume this is for "invisible" stuff that's not a wall.

Comment (mjrmua) [reply src/game.js]: If your visual range is <=0

File: src/visibility.js (hunk 2 above)
Comment (samperumal) [reply src/visibility.js]: Yeah, I reckon let's ensure all files end with newlines.

**Pull request merged**